lib/ohai/plugins/packages.rb

```diff
@@ -22,63 +22,78 @@
   provides "packages"
   depends "platform_family"
 
+  WINDOWS_ATTRIBUTE_ALIASES = {
+    "DisplayVersion" => "version",
+    "Publisher" => "publisher",
+    "InstallDate" => "installdate"
+  }
+
   collect_data(:linux) do
-    if configuration(:enabled)
```

Comment:
OK I think I have the proper background on why this plugin was disabled. It seems safe to enable it again. :)